src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute/Cmdlets/SetAzureRmMlOpCluster.cs

```diff
@@ -0,0 +1,199 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.MachineLearningCompute.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.MachineLearningCompute;
+using Microsoft.Azure.Management.MachineLearningCompute.Models;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.MachineLearningCompute.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Set, CmdletSuffix, SupportsShouldProcess = true)]
```

Comment:
@shutchings make sure that this cmdlet has a `DefaultParameterSetName`, which will be the `SetByIndividualParameters` set so users are prompted to provide `Name` and `ResourceGroupName`

Reply:
Sounds good.